stdlib/public/core/Policy.swift

```diff
@@ -104,7 +104,7 @@ public typealias StringLiteralType = String
 // IEEE Binary64, and we need 1 bit to represent the sign.  Instead of using
 // 1025, we use the next round number -- 2048.
 public typealias _MaxBuiltinIntegerType = Builtin.Int2048
-#if (!os(Windows) || CYGWIN) && (arch(i386) || arch(x86_64))
+#if os(Cygwin) && (arch(i386) || arch(x86_64))
```

Comment:
Just drop `CYGWIN`.
`#if !os(Windows) && (arch(i386) || arch(x86_64))`

Linux i386/x86_64, Cygwin i386/x86_64 and macOS x86_64 have `Builtin.FPIEEE80`.
Windows i386/x86_64 and other CPUs do not have.